Formula/gromacs.rb

```diff
@@ -1,27 +1,37 @@
 class Gromacs < Formula
   desc "Versatile package for molecular dynamics calculations"
   homepage "http://www.gromacs.org/"
-  url "https://ftp.gromacs.org/pub/gromacs/gromacs-2019.1.tar.gz"
-  sha256 "b2c37ed2fcd0e64c4efcabdc8ee581143986527192e6e647a197c76d9c4583ec"
-
-  bottle do
-    sha256 "94d33c3b2aab69176bb9ae38225a4486f8e648e775628046cc376d6110f70c64" => :mojave
-    sha256 "f5e98222c231b867623809636fe56ec550bf533f517b7312bb554a915bdf4fe9" => :high_sierra
-    sha256 "c4479eb9a92d1615c2247c9ba3ab185de46a35cc37b55c2a6443f0f73eb7c0cb" => :sierra
-  end
+  url "https://ftp.gromacs.org/pub/gromacs/gromacs-2019.3.tar.gz"
+  sha256 "4211a598bf3b7aca2b14ad991448947da9032566f13239b1a05a2d4824357573"
 
   depends_on "cmake" => :build
   depends_on "fftw"
-  depends_on "gsl"
+  depends_on "gcc" # for OpenMP
 
   def install
+    # Non-executable GMXRC files should be installed in DATADIR
     inreplace "scripts/CMakeLists.txt", "CMAKE_INSTALL_BINDIR",
                                         "CMAKE_INSTALL_DATADIR"
+    # fix an error on detecting CPU. see https://redmine.gromacs.org/issues/2927
```

Comment:
Upstream ticket says:

> I checked newer versions of CMake, for me the issue is gone

so we shouldn't hack that.

Reply:
OK. Please report it upstream, so they know it's not fixed and can investigate further.